pkg/datapath/connector/ipvlan.go

```diff
@@ -225,11 +225,13 @@ func CreateAndSetupIpvlanSlave(id string, slaveIfName string, netNs ns.NetNS, mt
 		return nil, fmt.Errorf("unable to setup ipvlan slave in remote netns: %w", err)
 	}
 
-	mapID, err := m.ID()
+	info, err := m.Info()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get map ID: %w", err)
+		return nil, fmt.Errorf("failed to get map info: %w", err)
+	}
+	if mapID, ok := info.ID(); ok {
```

Comment:
```suggestion
	mapID, ok := info.ID()
	if !ok {
	    return nil, fmt.Errorf("failed to get map ID")
	}
	ep.DatapathMapID = int64(mapID)
```
Maybe we should keep returning an error if the ID is not available, considering previous behaviour would return an error then?

Reply:
Thanks, fixed.